services/community-site/src/pages/Delegation/Delegation.tsx

```diff
@@ -334,7 +345,16 @@ const Delegation = () => {
                   delegatableNodes.map((row) => (
                     <TableRow className="tableRow">
                       <TableCell className="tableCell">{row.status}</TableCell>
-                      <TableCell className="tableCell">{row.name}</TableCell>
+                      <TableCell className="tableCell">
+                        <div className="flexCell">
```

Comment:
I think the icon is an inline element so no need to add it in a div to make it a block element and then to add flex over the name and icon

Reply:
As I said on Slack:
```
My idea of adding a flexbox is just to add space-between the trophy icon and the title of the node. I’ve added the legend.
Using flex is probably the easier thing here since there is some space between both elements (title and icon). Do you agree or could you suggest other approaches?
```
![imagen](https://user-images.githubusercontent.com/602693/148444835-7dff8136-9194-4247-bed9-060717f0fc3c.png)